stable/minio/values.yaml

```diff
@@ -123,14 +123,34 @@ gcsgateway:
   # Google cloud project-id
   projectId: ""
 
+## Use minio on NAS backend
+## https://docs.minio.io/docs/minio-gateway-for-nas
+
+nasgateway:
+  enabled: false
+  # Number of parallel instances
+  replicas: 4
```

Comment:
Why would it make sense to have that under `nasgateway` and only apply if `nasgateway.enabled` is `true`?